### ISSUE TITLE
questions about POKT vs. Covalent and where to place Covalent related scripts

### ISSUE THREAD

**Issue by sarob** [OpenDataforWeb3/Resources#43]
Expanding on the similar concept of the contributing doc just merged. I’ll dig into the work topics a bit to get a sense of effort required.

Ideally we can carry the hackathon contributors and sponsors over into the community. There will need to be some workflow before, during, and after.

**Comment by sarob** [comment]
Related to this issue, From @stefi-says:
Hey guys, this week I was focused in understand some workflows that participants could take during the hackathon and one of them was network analysis. I did some research ( <@968390919470608384> even tried to help me ) and seems Pokt does not have an endpoint that returns transactions indexed by wallet addresses ( at least for ethereum and polygon) .  This could diminished the number of submissions using Raphtory ( for network analyses) and analysis for the contest #3  . 
Happily in our landscape we have Covalent that makes that endpoint available 🙂  . Long history short, I made an python script to help participants easily get data from their API, I was wondering:
- how should I make this available? At "resources" on our website? ( will write some explanations on where to get API key and documentation)
- I notice that on the website the "resources page " is different from the landscape, actually Covalent is not present on the website as it is on the landscape. Is it intentional?

**Discord msg posted https://discord.com/channels/1037443230993743902/1048288174805352530/1058550375885578400**
